### PR TITLE
5 provide short description of space

### LINKED ISSUE #5: Provide short description of space
```
As a user
So that I can list my property's features, 
I want to be able to provide a short description
```

### PR DESCRIPTION
**Pull request by irynahowarth**
closes #5

### CODE REVIEW

File: spec/web_helpers.rb [hunk 1]
```diff
@@ -1,7 +1,8 @@
-def create_space
+def create_space(name, description="")
```
Comment (RPiper93) [inline spec/web_helpers.rb]: Maybe the default description should be "description" instead of an empty string?

Comment (tcpickard94) [inline spec/web_helpers.rb]: We did this as we didn't make description as a required field. Do you think it needs to be?

Comment (RPiper93) [inline spec/web_helpers.rb]: Probably? I mean, you wouldn't want to book a BnB without any sort of description, would you?

**Comment by RPiper93** [comment]
:shipit:

**Pull request merged**